rsmanage/rsmanager.py

```diff
@@ -385,14 +385,17 @@ def env(config, checkdb):
         else:
             dbinit = 2
             print("Database is initialized")
-    if os.path.exists(DBSDIR):
-        dbdir = 1
-        print("Database migration folder exists")
-    else:
-        dbdir = 0
-        print("No Database Migration Folder")
 
-    echoEnviron(config)
+        if os.path.exists(DBSDIR):
+            dbdir = 1
+            print("Database migration folder exists")
+        else:
+            dbdir = 0
+            print("No Database Migration Folder")
+
+    if not checkdb or config.verbose:
+        echoEnviron(config)
```

Comment:
Is this going to print passwords / etc. in clear sight? You have to be careful with docker-compose / docker - all of these printed statements will be (forever, pretty much) in the logs.